### DISCUSSION TITLE
Totally brillant thanks!

### DISCUSSION THREAD

**Discussion by Sausages69** [feelfreelinux/octo4a#304] [General]
What a great idea and piece of software! Especially these days since Pi's are expensive or unobtainable.
This is now my daily Octoprint instance due to the camera support.

Just need Homeassistant on Android next thanks ;-)

**Comment by mordenm1** [comment]
I'm going to add a +1 on this I have been struggling on getting octoprint to run on a macbook to work for a week and a half now. Installed this on an old Galaxy S8 with a cracked screen and worked perfectly. Cant believe how seamless it is.
